### PR TITLE
Add `bugs-tab` feature

### LINKED ISSUE #2749: Add "Bugs" tab
It would strictly link to the [`bug` label](https://github.com/sindresorhus/refined-github/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug), which is the most common label name. The "Pinned issues" section would be hidden like it's hidden in the PR tab.

If no issues are found under the label, the tab wouldn't be added.

Useful or annoying? I feel that the current "Issues" tab hides bugs that need to be fixed by merging them with feature requests and all kinds of other noise.

### PR DESCRIPTION
**Pull request by fregante**
Closes #2749 

<table><td>bugs-tab</table>

Your thoughts needed on the ❓ tasks:

- [x] Basic implementation
- [x] Cache ~~but prefer local count if available~~ (edit: way too convoluted)
- [x] Subtract bugs from Issues tab’s count ❓ 
- [x] Hide bugs from Issues tab ❓ 
- [x] Improve with #2752
- [x] Add some comments and document

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -204,6 +204,7 @@ Thanks for contributing! 🦋🙌
 - [](# "cycle-lists-with-keyboard-shortcuts") [Allows the <kbd>↑</kbd> and <kbd>↓</kbd> keys to cycle "popover lists" (labels, milestones, etc).](https://user-images.githubusercontent.com/37769974/59158786-6fd2c400-8add-11e9-9db1-db80186fa6ea.gif)
 - [](# "toggle-everything-with-alt") [Adds a shortcut to toggle all similar items (minimized comments, deferred diffs, etc) at once: <kbd>alt</kbd> <kbd>click</kbd> on each button or checkbox.](https://user-images.githubusercontent.com/37769974/62208543-dcb75b80-b3b4-11e9-984f-ddb479ea149d.gif)
 - [](# "quick-mention") [Adds a button to `@mention` a user in discussions.](https://user-images.githubusercontent.com/1402241/70406615-f445d580-1a73-11ea-9ab1-bf6bd9aa70a3.gif)
+- [](# "bugs-tab") [Adds a "Bugs" tab to repos, if there are any open issues with the "bugs" label.](https://user-images.githubusercontent.com/1402241/73720910-a688d900-4755-11ea-9c8d-70e5ddb3bfe5.png)
```
Comment (FloEdelmann) [inline readme.md]:
```suggestion
- [](# "bugs-tab") [Adds a "Bugs" tab to repos, if there are any open issues with the "bug" label.](https://user-images.githubusercontent.com/1402241/73720910-a688d900-4755-11ea-9c8d-70e5ddb3bfe5.png)
```

File: source/features/bugs-tab.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		// Don't await; if there are no pinned issues, this would delay the bug count update
+		elementReady('.js-pinned-issues-reorder-container').then(pinnedIssues => pinnedIssues?.remove());
+	}
+
+	select('.Counter', bugsTab)!.textContent = String(await countPromise);
+}
+
+features.add({
+	id: __featureName__,
+	description: 'Adds a "Bugs" tab to repos, if there are any open issues with the "bugs" label.',
```
Comment (FloEdelmann) [inline source/features/bugs-tab.tsx]:
```suggestion
	description: 'Adds a "Bugs" tab to repos, if there are any open issues with the "bug" label.',
```

**Comment by fregante** [comment]
Thank you!

**Comment by staabm** [comment]
would it make sense to have something similar for 'good first issues'?

**Comment by fregante** [comment]
I don’t think that’s nearly as useful for most people. I kinda have an idea for a “customizable tabs” extension but not as part of Refined GitHub

**Comment by fregante** [comment]
I made the last change: exclude `label:bug` from Issues.

I fear this feature will be controversial, but then again it can be disabled. I'll probably still extract it into its own configurable/smarter extension at some point.

**Comment by sindresorhus** [comment]
> I made the last change: exclude label:bug from Issues.

Makes sense.

**Comment by fregante** [comment]
Thanks!

**Comment by staabm** [comment]
Since we have this bugs tab, it might make sense to additionally have a „thank you“ tab, which lists apprecitations from the community?

https://twitter.com/dzhavatushev/status/1241619880736182273?s=20

**Pull request merged**